src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -95,7 +97,7 @@ class MetadataBasedItemList extends React.Component<Props, State> {
             isUpdating: false,
             scrollLeftOffset: 0,
             scrollRightOffset: 0,
-            valueBeingEdited: null,
+            valueBeingEdited: undefined,
```

Comment:
Changing from `null` to `undefined` to make use of default `propTypes` in components deep down in the component tree.

Reply:
would you want to remove `valueBeingEdited` from here then, since its `undefined`?